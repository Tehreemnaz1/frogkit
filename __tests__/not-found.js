import { mount } from 'vue-test-utils'
import NotFound from '@/components/not-found'

describe('Not Found component', () => {
  it('should render component and match snapshot: lang de', () => {
    const wrapper = mount(NotFound, {propsData: {lang: 'de'}})
    expect(wrapper.html()).toMatchSnapshot()
  })
  it('should render component and match snapshot: lang en', () => {
    const wrapper = mount(NotFound, {propsData: {lang: 'en'}})
    expect(wrapper.html()).toMatchSnapshot()
  })
})