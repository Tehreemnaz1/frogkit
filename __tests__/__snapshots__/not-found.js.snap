// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Not Found component should render component and match snapshot: lang de 1`] = `
<flex align="center" justify="center" column="" class="fk-not-found">
    <icon icon="exclamation-circle" color="orange" class="fk-not-found__exclamation m-b-30"></icon>
    <title color="primary" size="xl" class="m-b-40">Ein Fehler ist aufgetreten</title>
    <p class="fk-not-found__content m-b-40">
        Es tut uns leid, wir konnten die angeforderte Seite nicht finden. <br> Bitte die Internetadresse (URL) auf Fehler überprüfen. <br></p> <button href="#" size="big" color="secondary" link="" outline="" class="fk-not-found__button">
    Zur Startseite <icon icon="arrow-right" color="secondary"></icon></button></flex>
`;

exports[`Not Found component should render component and match snapshot: lang en 1`] = `
<flex align="center" justify="center" column="" class="fk-not-found">
    <icon icon="exclamation-circle" color="orange" class="fk-not-found__exclamation m-b-30"></icon>
    <title color="primary" size="xl" class="m-b-40">An error has occured</title>
    <p class="fk-not-found__content m-b-40">
        We're sorry, we couldn't find the page you requested. <br> Please check the internet address (URL) for errors. <br></p> <button href="#" size="big" color="secondary" link="" outline="" class="fk-not-found__button">
    Go to homepage <icon icon="arrow-right" color="secondary"></icon></button></flex>
`;
